processing/src/main/java/io/druid/query/metadata/SegmentAnalyzer.java

```diff
@@ -161,28 +179,30 @@ private ColumnAnalysis analyzeNumericColumn(
         capabilities.hasMultipleValues(),
         size,
         null,
+        null,
+        null,
         null
     );
   }
 
   private ColumnAnalysis analyzeStringColumn(
```

Comment:
`column` is null for incremental index, which make impossible to use other impl.